File: C Programming/sum_of_digits.c
#include<stdio.h>

void main()
{
	int no, sum=0;
	int digit;
	printf("Enter the No\n");
	scanf("%d",&no);
	while(no > 0)
	{
		digit = no % 10;
		sum = sum + digit;
		no = no / 10;
	}
	printf("The Sum of Digits:%d\n",sum);
}

File: Java Programs/Sum_of_digits.java
import java.util.Scanner;

class Sum_of_digits
{
	public static void main(String[] args)
	{
		int no, digit, sum=0;
		Scanner scan = new Scanner(System.in);
		no = scan.nextInt();

		while(no>0)
		{
			digit = no % 10;
			sum = sum + digit;
			no = no / 10;
		}
		System.out.println("Sum of the Digits "+sum);
	}
}